service/common/src/main/java/org/apache/polaris/service/auth/DefaultOAuth2ApiService.java

```diff
@@ -114,8 +116,8 @@ public Response getToken(
     return Response.ok(
             OAuthTokenResponse.builder()
                 .withToken(tokenResponse.getAccessToken())
-                .withTokenType(OAuth2Constants.BEARER)
-                .withIssuedTokenType(OAuth2Properties.ACCESS_TOKEN_TYPE)
+                .withTokenType(BEARER)
+                .withIssuedTokenType(tokenResponse.getTokenType())
```

Comment:
nit: this looks like a functional change, not just "remove references to hadoop", even though it does make sense :) maybe rephrase the PR subject line?

Reply:
Done, let me know if that's better.

Reply:
sweet :+1: thx :)